src/utils/summary.cpp

```diff
@@ -346,6 +366,7 @@ void lbann_summary::gather_scalar_summary(
 void lbann_summary::gather_scalar_summary(const std::string tag,
                                           float s,
                                           int step) {
+#if 0
```

Comment:
Why is this preprocessed out? If it shouldn't be there, we should just delete the code. If it should be there, it shouldn't be preprocessed out. If it should conditionally be there, we should have a better condition than `if 0`...

Reply:
Upon further review, this code is needed and the preprocessor stuff will be deleted.